src/extensions/bigbang/bigbang.go

```diff
@@ -151,7 +151,7 @@ func Run(YOLO bool, tmpPaths types.ComponentPaths, c types.ZarfComponent) (types
 	for _, hrNamespacedName := range namespacedHelmReleaseNames {
 		hr := hrDependencies[hrNamespacedName]
 		action := types.ZarfComponentAction{
-			Description:     fmt.Sprintf("Big Bang Helm Release `%s` to be ready", hr.Metadata.Name),
+			Description:     fmt.Sprintf("Big Bang Helm Release `%s` to be ready (Namespace: %s)", hr.Metadata.Name, hrNamespacedName),
```

Comment:
```suggestion
			Description:     fmt.Sprintf("Big Bang Helm Release `%s` to be ready", hrNamespacedName),
```
Because `hrNamespacedName` already contains the HR name we could just use it instead.  I would be fine with the above _but_ we could also add a warning on the lookup of `hr := hrDependencies[hrNamespacedName]`.  If it was `hr, ok := hrDependencies[hrNamespacedName]` the `ok` boolean could be checked like below:

```
hr, ok := hrDependencies[hrNamespacedName]
if !ok {
message.Warnf("Discovered a missing dependent Helm release `%s`, reconfigure your Big Bang values file to add it.", hrNamespacedName)
}
```